introduction.md

```diff
@@ -3,4 +3,8 @@ JavaScript Introduction
 ==========
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?